deepchem/molnet/load_function/load_dataset_template.py

```diff
@@ -0,0 +1,202 @@
+"""
+Short docstring description of dataset.
+"""
+import os
+import logging
+import deepchem
+from deepchem.feat import Featurizer
+from deepchem.trans import Transformer
+from deepchem.split.splitters import Splitter
+
+from typing import List, Tuple, Optional
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_DIR = deepchem.utils.get_data_dir()
+MYDATASET_URL = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/mydataset.tar.gz'
+MYDATASET_CSV_URL = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/mydataset.csv'
+
+# dict of accepted featurizers for this dataset
+DEFAULT_FEATURIZERS = {
```

Comment:
Yes, definitely! I was thinking it would be nice to have a centralized list of all the allowed featurizers, transformers, and splitters so that the string keys referring to each one are standard. The template is a good place for that.

I could set up simple kwargs to enable `NormalizationTransformer` and any other transformers that require a dataset to initialize.

Reply:
Good idea! Perhaps we should make a new file `dc.molnet.defaults` that maintains the centralized list of defaults? We can point to it in the docs and use it in the template and different loader files?

Reply:
That sounds good, it will be easier to maintain in a new file. I'll make that file and update the docs!

Reply:
I added a `dc.molnet.defaults` module with a `get_defaults` utility function that inspects the featurizer, transformer, and splitter modules to generate a dict of all the available classes. There's also a saved `deepchem/molnet/defaults.json` file that is referenced in the docs.